force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -83,6 +113,15 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Even" or "Odd", or null if the number is null.
      */
     public static String checkEvenOdd(Integer a) {
+        if (a == null) {
+            return null;
+        }
+        else if(Math.mod(a, 2) == 0){
+            return 'Even';
+        }
+        else if (Math.mod(a,2) != 0) {
+            return 'Odd';
+        }
         return null; // Replace null with the variable you used to store the result
     }
```

Comment:
Returning null is always risky can can cause bugs. Could you use an else statement instead?